poetry.lock

```diff
@@ -0,0 +1,806 @@
+[[package]]
```

Comment:
Since this is a library, add the `.lock` file to .gitinore.

From Poetry docs:

> For libraries it is not necessary to commit the lock file.

https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control

---

Here is another source about when you should keep lockfiles in (or out) of version control:
https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries